src/main/java/frc/robot/RobotContainer.java

```diff
@@ -186,15 +213,65 @@ private void configureBindings() {
         // Engage        
         controller.rightBumper().onTrue(CommandUtil.logged(new EngageRatchet(climber)));
 
-//        controller.a().onTrue(new StartFeeder(feeder));
-//        controller.b().onTrue(new ExitAndShoot(shooter,feeder));
-//        ramp.setDefaultCommand(new RampMove(ramp, 10));
-//        climber.setDefaultCommand(new ManualClimb(climber, controller::getLeftX));
-        controller.a().onTrue(CommandUtil.logged(new StartIntakeAndFeeder(feeder,intakeSubsystem,deployer,ramp)));
-        controller.b().onTrue(CommandUtil.logged(new ExitAndShoot(shooter,feeder, drivetrain)));
-        controller.x().onTrue(CommandUtil.logged(new LowerDeployer(deployer)));
-//        controller.a().onTrue(new DeployerLower(deployer));
-//        controller.b().onTrue(new DeployerRaise(deployer));
+        // Set up to shoot Speaker CLOSE - Y
+        controller.y().onTrue(CommandUtil.parallel("Setup Speaker Shot (CLOSE)",
+                new RampMove(ramp, () -> GameConstants.RAMP_POS_SHOOT_SPEAKER_CLOSE),
+                new ShootSpeaker(shooter, drivetrain)));
+
+        // Set up to shoot Speaker AWAY - X
+        controller.x().onTrue(CommandUtil.parallel("Setup Speaker Shot (AWAY)",
+                new RampMove(ramp, () -> GameConstants.RAMP_POS_SHOOT_SPEAKER_AWAY),
+                new ShootSpeaker(shooter, drivetrain)));
+
+        // Set up to shoot AMP - A
+        controller.a().onTrue(CommandUtil.parallel("Setup Amp shot",
+                new DeployAmp(amp),
+                new RampMove(ramp, () -> GameConstants.RAMP_POS_SHOOT_AMP),
+                new ShootAmp(shooter)));
+
+        // Shoot the note - B
+        controller.b().onTrue(CommandUtil.sequence("Operator Shoot",
+                new FeederGamepieceUntilLeave(feeder),
+                new WaitCommand(GameConstants.SHOOTER_TIME_BEFORE_STOPPING),
+                new StopShooter(shooter),
+                new RetractAmp(amp)));
+                new RampMove(ramp, () -> GameConstants.RAMP_POS_STOW);
+
+        joyRightButton2.onTrue(CommandUtil.sequence("Driver Shoot",
+                new FeederGamepieceUntilLeave(feeder),
+                new StopShooter(shooter),
+                new RetractAmp(amp)));
+
+        // amp up and down
+        controller.povLeft().onTrue(CommandUtil.logged(new ToggleAmp(amp)));
```

Comment:
Would it be better to make this explicitly "left to deploy amp" / "right to retract amp" ?